src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -275,8 +276,15 @@ class Manager : public DoutPrefixProvider {
           << " failed. error: " << ret << " (will retry)" << dendl;
         return EntryProcessingResult::Failure;
       } else {
-        ldpp_dout(this, 20) << "INFO: push entry: " << entry.marker << " to endpoint: " << event_entry.push_endpoint 
-          << " ok" <<  dendl;
+        ldpp_dout(this, 1) << "INFO: push entry: " << entry.marker
```

Comment:
IMO, we shoudl not log info at debug level 1 - should be used for errors

Reply:
> IMO, we shoudl not log info at debug level 1 - should be used for errors

yes i understand your concern, but i wrote in the commit message the whole reason on why i am turning this message on by default.
Currently there is not metric/log that would tell about the event delivered successfully, similar to S3(beast) log which logs each request with the success/failure. Having this line there helps some transparency on whether the event was delivered successfully.
If your concern is only about the INFO message getting logged at level 1, i can remove the INFO from the log line ?

Reply:
my concern is flooding the log, and increasing cpu consumption because of that.
if we are missing notification metrics, lets add them

Reply:
> my concern is flooding the log, and increasing cpu consumption because of that. if we are missing notification metrics, lets add them

adding metric is not trivial, as the current metric is more generic , does not even have it per/topic

Reply:
sadly, logging is one of the most impacting factors on rgw cpu consumption.

Reply:
So do we have any other option ?
Is it gonna be tooo spammy ?

Reply:
> So do we have any other option ? 

lets implement per topic labeled stats

> Is it gonna be tooo spammy ?

IMO, yes

Reply:
As discussed during the rgw_meeting removing the log line here and instead making it part of rgw_notification_subsystem for logging and will put that as part of different PR, so reverting this commit